packages/editor-ui/src/components/WorkflowSelectorParameterInput/useWorkflowResourcesLocator.ts

```diff
@@ -24,8 +24,6 @@ export function useWorkflowResourcesLocator(router: Router) {
 	);
 
 	const filteredResources = computed(() => {
-		if (!searchFilter.value) return workflowsResources.value;
```

Comment:
Could not find why this was needed, but all the e2e tests are passing so we should be fine without this line.

Reply:
Can you double-check with the original author about it?

Reply:
@OlegIvaniv do you remember the reason for this line? 🤔 I had to remove this line because once the sub workflow is created, and we pull all the workflows again, we need to re-render the list. Apparently this does not break the existing behavior.

Reply:
This was an optimization to return the already formatted `workflowsResources`. Because those are in the mapped format(`Array<{ name: string; value: string; url: string }>` so there's no point in mapping them again with `workflowDbToResourceMapper`. The problem with the workflows not updating is because we're calling `setWorkflowsResources` on component mount and the input doesn't remount when you create a new workflow. To always pull the latest list of workflows we'd need to do something like:
```ts
watch(isDropdownVisible, (isVisible) => {
	if (isVisible) {
		void setWorkflowsResources();
	}
});
```

Reply:
No, no, the effect definitely won't be so drastic. Happy with this approach too.
Awesome feature btw! Will save me so much time :D